pkg/graph/email.graphqls

```diff
@@ -77,11 +77,25 @@ type DomainConnection {
     pageInfo: PageInfo!
     totalCount: Int!
 }
+type GroupConnection {
+    edges: [Group]
+    pageInfo: PageInfo!
+    totalCount: Int!
+}
 
 input EmailAccountFilter {
     type: EmailAccountType
 }
 
+type Group {
```

Comment:
I don't want to use `group` as a name for email groups. We should prefix all email related objects with `email`. For example, we might want to have school groups and something else. So to make it less ambiguous we should prefix groups. My suggestion would be to use the actual `emailAccounts` object. Maybe there's a way to make it work with that object? GraphQL is quite versatile.

In case we cannot get it to work, we should name it `emailGroup`.